backend/src/bundles/users/user.service.ts

```diff
@@ -43,14 +60,26 @@ class UserService implements Service {
         };
     }
 
-    public async create(payload: UserSignUpRequestDto): Promise<UserEntity> {
-        return this.userRepository.create(
+    public async create(
+        payload: UserSignUpRequestDto,
+        // TODO: change UserGetOneItemResponseDto to type for user.toObject()
```

Comment:
disagree! if I want to return only a few fields?

Reply:
Oh, really. We can manage the params that return